conjureup/utils.py

```diff
@@ -24,6 +25,15 @@
 from conjureup.telemetry import track_event
 
 
+def is_snap_package_installed(pkg):
+    # snapd is not idempotent so we need to query first
+    snap_query = 'http+unix://%2Frun%2Fsnapd.socket/v2/snaps/{}'.format(
+        pkg)
+    with requests_unixsocket.Session() as session:
+        if session.get(snap_query).ok:
+            return
+
```

Comment:
This function always returns None. as used, this is equivalent to always False - I'd suggest making it explicitly return True and False as appropriate.